main/py-pbr/APKBUILD

```diff
@@ -7,24 +7,49 @@ pkgrel=0
 pkgdesc="Python Build Reasonableness"
 url="https://pypi.python.org/pypi/pbr"
 arch="noarch"
-license="APACHE2.0"
-depends="python2"
-depends_dev=""
-makedepends="$depends_dev python2-dev py-setuptools"
-install=""
-subpackages="$pkgname-dev"
+license="ASL 2.0"
```

Comment:
I'm not sure where else the license information is used but the package DB seems to grab the full entry. Talking about separation...`MIT BSD` and `GPLv2 and GPLv2+ and GPLv3 and MIT` :wink: